docs/src/main/sphinx/connector/kafka.rst

```diff
@@ -420,6 +422,19 @@ table description supplier are:
 * There is no need to define tables manually.
 * Some Protobuf specific types like ``oneof`` are supported and mapped to JSON.
 
+When using Protobuf decoder with the Confluent table description supplier, the
```

Comment:
I thought this over some more and checked with other docs .. we should move all this up to the requirements section and then link to it from here

Reply:
Moved new material to requirements section and linked.